pos_backend_partner/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+POS Backend Partner
+===================
+
+Choose a POS customer from the backend.
+
+
+- Replace customer selection screen in POS by a backend view.
+- Perf improvement : Full customer list is not downloaded anymore by the POS at startup.
```

Comment:
But this actually means that POS will not work offline. I don't think that we use work "improvement" here. It's a "change" or something like that

Reply:
Well, it's a performance improvement, but at the expense of offline mode

Reply:
Just found, that a note was added to base module:
https://github.com/OCA/pos/pull/226/commits/946882de2e503c3950c148a8309d386b1cb552a4

Still think that it has to be added here too

Reply:
It's a huge perf improvement if you have a lot of customers. In my case it's about 200 000 partners, and users have browsers warnings about slow and unresponsive scripts.

Offline mode still works, you just loose the ability to have a named customer while offline.

Readme has been corrected with your remarks.

Reply:
Analog of pos_cache with customers caching may resolve this problem?

Reply:
When you introduce cache, you add a lot of complexity. 
Here it's much simplier : no list, nothing to wait, nothing to invalidate, always up to date.

Perfs is one nice part of the features, having advanced search and view customisations in one place reduce dev time.